inspirehep/modules/workflows/workflows/article.py

```diff
@@ -317,26 +262,180 @@
 ]
 
 
-ADD_MARKS = [
+STOP_IF_EXISTING_SUBMISSION = [
     IF(
-        article_exists,
+        is_submission,
+        IF(
+            is_marked('is-update'),
+            NOTIFY_ALREADY_EXISTING
+        )
+    )
+]
+
+
+HALT_FOR_APPROVAL = [
```

Comment:
Remove this (or CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL), and maybe rename.

Reply:
removed CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL